system-test/system.ts

```diff
@@ -1,17 +1,32 @@
-/**
- * Copyright 2018 Google LLC. All Rights Reserved.
+/*
+ * Copyright 2019 Google LLC
+ * All rights reserved.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are
+ * met:
  *
- *      http://www.apache.org/licenses/LICENSE-2.0
+ *     * Redistributions of source code must retain the above copyright
+ * notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following disclaimer
+ * in the documentation and/or other materials provided with the
```

Comment:
I just realized the whole gax project is BSD-licensed, and some files I added later are Apache-licensed. Since they were no changes from external contributors in those files (I actually checked), it should be OK to relicense them as BSD to match other files in the project (I recently saw this discussion elsewhere, that's what we were told then).